docs/source/contribution.rst

```diff
@@ -37,17 +37,20 @@ branches based off of development and merged into it once stable and
 reviewed. To submit code, follow these steps:
 
 1. Create a new branch off of development. Select a descriptive branch
-   name. We highly encourage you to use `autopep8` to follow the PEP8 styling. Run the following command before creating the pull request:
-
+   name. 
    ::
-
-       autopep8 --in-place --exclude venv,docs --recursive .
-
        git fetch upstream
        git checkout master
        git merge upstream/master
        git checkout -b your-branch-name
 
+   We highly encourage using `black <http://www.github.com/psf/black>`_
+   to format your code. It sticks to PEP8 and is in line with the rest
+   of the repo. To use black, install it via `pip` and run the following
+   command:
+   ::
```

Comment:
@nikochiko  Can you please add the usage of precommits as mentioned by @RishabhJain2018 https://github.com/Cloud-CV/EvalAI/pull/2762#issuecomment-620754957

Reply:
But that's strange. The hooks are already set up via `pre-commit` but I didn't get results of flake8 and black didn't format my very inconsistent test file while testing.

Reply:
Added! :+1: 
@Ayukha thanks for the pointer.